main.go

```diff
@@ -44,6 +44,7 @@ Logging Options:
     -D, --debug                      Enable debugging output
     -V, --trace                      Trace the raw protocol
     -DV                              Debug and trace
+    --sys_trace                      Trace the system account (default: false)
```

Comment:
I like it from some tools that they use `-vv` and `-vvv` to define "very verbose" and extra verbose levels on the command line, (examples I can think of are memcached and curl), maybe could be a good fit for the nats-server too? Could be in this case that `-VV` means very verbose, activating tracing of system events for example

Reply:
How about adding -VV Very verbose, trace and sys_trace

Reply:
-VV and -DVV, config option is called trace_verbose.